style.css

```diff
@@ -252,3 +264,176 @@ textarea:focus {
   padding: 5px;
   background-color: #60c095;
 }
+
+/* Desktop version */
+@media only screen and (min-device-width: 768px) {
+  body {
```

Comment:
- [ ] The experience I had when viewing your website with this rule was not good and felt like it was desktop first instead of it being a mobile-first. But When I changed it to the required rule below, the experience was great and all your styles were intact.
- I suggest you replace your rule with this `@media screen and (min-width: 750px)` instead of the one you're having `@media only screen and (min-device-width: 768px)`.